besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -831,6 +831,14 @@ void setBannedNodeIds(final List<String> values) {
       arity = "1")
   private final Wei txFeeCap = DEFAULT_RPC_TX_FEE_CAP;
 
+  @Option(
+      names = {"--rpc-require-chainid-in-txs"},
+      description =
+          "Allow for unprotected (non EIP155 signed) transactions to be submitted via RPC (default: ${DEFAULT-VALUE})",
+      arity = "1")
+  // TODO: set default to false for next major release
+  private final Boolean unprotectedTransactionsAllowed = true;
```

Comment:
This looks inverted to me: if `--rpc-require-chainid-in-txs=true`, then shouldn't `unprotectedTransactionsAllowed == false`?  Suggest changing this variable to `requireTxReplayProtection` or similar.  I think the flag name could be clearer as well, maybe: `--require-tx-replay-protection`.

Reply:
That sounds easier to parse indeed. I'll change to your suggestion.